screens/HomeScreen.js

```
@@ -134,6 +135,8 @@ const HomeScreen = () => {
 		}
 	}
 
+	const castButtonTopOffset = rootStore.isFullscreen ? 0 : (Platform.OS === 'ios' ? insets.top : (safeAreaPadding.paddingTop ?? insets.top));
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)